Octokit.Reactive/Clients/IObservablePackagesClient.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Reactive;
+
+namespace Octokit.Reactive
+{
+    public interface IObservablePackagesClient
+    {
+        IObservablePackageVersionsClient PackageVersions { get; }
```

Comment:
So this is one of those weird API domains where package versions are a subset of packages, yet a package version is a package.

What you've done here feels like it works - I'm glad you approached the relationship this way to help reduce namespace confusion - I think leaving PackageVersions out of this interface would've made things a bit more clumsy.

```
github.Packages.PackageVersions.GetAllForUser
github.Packages.GetAllForOrg
```

feels more elegant than something like (though possible):

```
github.PackageVersions.GetAllForUser
github.Packages.GetAllForOrg
```

While, as you've implemented, it can lead to a bit more verbose API call, I think the expression feels more natural ❤️  solid work here!

Reply:
Thanks! It is a little odd. I took my inspiration from the GitHub Api docs where it's all grouped together in one Packages section.

I've just pushed a final fix for the typos also.